source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -221,10 +222,16 @@ void BioXASSideAppController::setupUserInterface()
 	onScalerConnected();
 
 	// Create the 32 element Ge detector view.
-	BioXAS32ElementGeDetectorView *geDetectorView = new BioXAS32ElementGeDetectorView(BioXASSideBeamline::bioXAS()->ge32ElementDetector());
-	geDetectorView->buildDetectorView();
-	geDetectorView->addEmissionLineNameFilter(QRegExp("1"));
-	geDetectorView->addPileUpPeakNameFilter(QRegExp("(K.1|L.1|Ma1)"));
+//	BioXAS32ElementGeDetectorView *geDetectorView = new BioXAS32ElementGeDetectorView(BioXASSideBeamline::bioXAS()->ge32ElementDetector());
+//	geDetectorView->buildDetectorView();
+//	geDetectorView->addEmissionLineNameFilter(QRegExp("1"));
+//	geDetectorView->addPileUpPeakNameFilter(QRegExp("(K.1|L.1|Ma1)"));
```

Comment:
I'm leaving the 32 element code commented out for the time being until it's up and running.